wolfssl/internal.h

```diff
@@ -1050,7 +1050,7 @@ enum {
 /* set maximum DH key size allowed */
 #ifndef WOLFSSL_MAX_DHKEY_BITS
     #ifdef WOLFSSL_MAX_STRENGTH
-        #define WOLFSSL_MAX_DHKEY_BITS 3072
+        #define WOLFSSL_MAX_DHKEY_BITS 4096
     #else
         #define WOLFSSL_MAX_DHKEY_BITS 2048
```

Comment:
let's keep the default max at 4096 so as not to break existing interop (independent of MAX_STRENGTH).

Reply:
Changed as required.